### ISSUE TITLE
Modification: Large Dataset Notary Process - Reducing time to DataCap for LDN

### ISSUE THREAD

**Issue by galen-mcandrew** [filecoin-project/notary-governance#217]
Preface: Support for large datasets on Filecoin Plus via the LDN process is still relatively nascent, and the intent with the original flow was to experiment with a set of initial clients up to 50PiB, collect feedback, iterate, and improve the flow. Based on current client experiences, this proposal attempts to reduce the friction for clients to receive the DataCap they need to onboard data onto Filecoin.

## Issue Description
Time to initial DataCap for large dataset clients is too slow at the moment: ~6 weeks to go from filing an application to receiving their first DataCap allocation. This is unsustainable/unscalable for real-world client use cases that need to use Filecoin at scale.

## Impact
A goal of Fil+ is to make the network more productive by enabling clients to make deals with less friction while reducing the liabily storage providers take on when making storage deals (increased compensation from the network, verification of clients by notaries). A process this slow makes it very difficult for a client to choose Filecoin as a storage solution since most alternatives do not require such a time-intensive up-front investment from a client.

## Proposed Solution(s)
**Main change:** All* notaries are added to each large dataset notary multisig at creation, and threshold for messages is reduced to 2. (**this will be presented to notaries who can then choose to opt-out of being included in this process*)

**Detailed process:**
1. Client submits LDN application in GitHub LDN Repo
1. Governance team + bot audits applications for basic completeness, not due diligence. Missing, incomplete, or invalid sections are redirected back to client.
1. Once the application is validated for completeness, bot creates multisig with all notaries as signors, with threshold set to 2
1. Bot proposes Notary status for multisig to RKH
1. RKH approve Notary status
1. Bot initiates request to LDN multisig for first allocation of DataCap
1. Notaries perform due diligence, and sign proposal via Fil+ Registry App; 2 signatures are required to approve DataCap allocation.

Process for subsequent allocations: 
1. A separate bot monitors DataCap available to a client and requests additional DataCap once the client address has used 75%+ of their previous DataCap allocation
1. Notaries perform subsequent due diligence on previous allocations
1. 2 notaries sign subsequent request

**Main differences between current and new process:**
1. LDN multisig is created with fewer manual steps after application is submitted (since due diligence happens later in the process)
1. First allocation request is kicked off without manual steps required from a client and the lead notary
1. Allocations only require 2 signatures rather than 4 - this decreases turnaround time for clients but still preserves the ability for notaries to carry out due diligence

**Additional potential considerations for discussion:**
1. For geo-decentralization, require that each of the 2 signatures come from notaries in different regions
1. For governance-decentralization, require that no Notary can approve 2 allocations from the same LDN multisig in a row
1. To ensure compliance with the program, the community should strictly enforce that notaries follow the process and use the requisite tooling. Non-compliance could result in revoking of notary status?

## Related Issues
https://github.com/filecoin-project/notary-governance/issues/94

**Comment by dannyob** [comment]
This looks mostly good to me. A few questions/suggestions:

1. Definitely approve of the "presumptively create notary first after low procedural hurdle, do due diligence during the actual datacap allocation."
2. Any reason for a two-notary requirement (as opposed to three, or keeping it at the current four)? I'd just like to get some insight into this figure. My hunch is that dropping to two might be too much of a relaxation to do in parallel with this change.
3. I feel like we should both approve geo-decentralization, but also ensure that there's some local knowledge being applied here. Maybe one notary from the region, one not? (Or whatever permutation you'd want for >2 notary sign-offs.) I'm aware this might add friction though, so not very committed to this. (Also, unclear how you actually automatically determine what the 'home" region is -- certainly a lot of the low-datacap requests I get are clearly from outside my region, or mislabel their region.)
4. We should definitely start a conversation about creating incentives/disincentives for notaries to follow a process, and not cut corners. But I think that's for a separate stage, after we've firmed up the process and have a clearer idea of where abuses/cut corners might be.

**Comment by MegTei** [comment]
I wont be on todays call (middle of the night down under), heres my input:
The proposed changes appear to create simplification. Will this also include RKH, seems things can stall for a while between notaries and RKH.
I have one further ask to be included in the process, Improved onboarding. Ref Internet Archive Issue #22  They have shared a lot of info about what they need from Filecoin and their success criteria, Is there tech sales or ecosystem lead to work through this with them so that we achieve a good experience (together).

**Comment by kernelogic** [comment]
I think lead notary should provide a document/record on due diligence performed on the client.

Also more contact information of notaries is needed. Some notaries never respond to github comment or slack DM so it's impossible to reach to them.

**Comment by Broz221** [comment]
I support that LDN process should be quicker and simpler. But I think this issue can not solve the problem fundamentally. It seems that clients only need 2 notaries instead of 4, but notaries should be different for each allocation. So eventually, also many notaries are needed. I would like to suggest that the allocation amount can be more to make this issue more reasonable. @galen-mcandrew   Let's discuss about it!

**Comment by Broz221** [comment]
As a notary of Estuary, I also found that the process was too long. Also, the reminding of multisig is not obvious enough to run it immediately (feel so sorry about Estuary). If issue 217 is adopted, Estuary need to find more notaries each time instead of only 4 notaries.

**Comment by MakaiRubio** [comment]
The process will be too slow in this way！Last time my case was influenced！I think you should improve it actually. It‘s necessary to make the step easier！

**Comment by swatchliu** [comment]
Dear All， 

please correct me if we are wrong.

According to #217, a 5PiB LDN allocation will be like the below sheet:
![001](https://user-images.githubusercontent.com/46444977/130463217-fcf52f08-c08b-464a-828f-667135454896.png)
27 weeks that's 189 days. Besides, these are the time cost for storage only, if we calculate the time cost of finding notaries(the Average time to first response from notary is 8D:19H) https://filplus.d.interplanetary.one/statistics   
Let's say 4 days is the average, that's 27 * 4=108 days. To complete the 5PiB storage needs 189+108=297 days. (Times are no including the data transfer, maintenance of the storage  machine) Such a time cost is unacceptable for both the miner and storage client. How come to spend 0.8 years making storage?

**What are we proposed:** 
1 No limited of the LDN weekly allocation.
2 Project also needs 7 notaries to evaluate, but 2 notaries multsign are enough.
3 Allocation depends on the miner's sealing capability(less than 80% of the weekly goal will reduce the 50% allocation of the goal.)
![002](https://user-images.githubusercontent.com/46444977/130463388-d4e957c1-11d2-493e-a22e-bfcada3f817b.png)
Let's check the time cost again.
![003](https://user-images.githubusercontent.com/46444977/130463511-1e62d734-b4d8-48d6-9084-f3733479ec6e.png)
7*14+4*14=154 days.  48% time costs are saving from this issue. If the notaries meeting can remind each notary to do the multsign in time, the time cost will be reduced a lot.

**Comment by ozhtdong** [comment]
@galen-mcandrew Hi, Galen. I have different opinions on this issue. First of all, I don't think only 2 notaries can approve an LDN application since the datacap amount of LDN is too large. But I understand your mood of reducing time, it's such a long process. And I think the main problem is the current allocation proportion. You can see such a long table above made by ByteBase. I sincerely hope that the first allocation proportion can remain, but forward allocation can be larger. Also, A&B, C&D is complicated. Maybe A&B can approve, C&D can stop. If any wrong, C&B have right to stop.

**Comment by baarenkassee07** [comment]
I agree with the view above. I think it greatly improve efficiency. We should reduce some time which is wasted on the process.

**Comment by kernelogic** [comment]
I agree with @ozhtdong about 2 notary to approve a LDN is too few. I think the subsequent allocation can be approved by just 2 notaries. But the initial due diligence should still be performed more thoroughly by more notaries.

**Comment by momack2** [comment]
Tagging @whyrusleeping since he also had some ideas to improve the process

**Comment by jnxc** [comment]
The weekly allocation model is too complicated, it can't match our real requirements.

**Comment by galen-mcandrew** [comment]
Thanks for all the great community discussion here! We'll be talking about this more in the upcoming Notary Governance calls on Tuesday Aug 31 ([Issue reminder](https://github.com/filecoin-project/notary-governance/issues/221))

I wanted to take a moment and address some of the comments here before that call.

As a reminder, the goal of this proposal is to help expedite the large dataset process, which is currently taking months. We will continue to evaluate this process, and we expect to make more modifications & proposals in the future.

---
**Concern:** *First DataCap allocation is too low*

Primarily, this proposal is to lower the initial barrier and time delay, to more quickly make medium-size first allocations (25-50TiB). While this is low in comparison to the full 5PiB a client may be requesting, the hope is to get a reasonably useful but lower risk amount of DataCap to a client quickly. Then notaries can leverage more tools to help audit the qualitative (written application) and quantitative (DataCap allocation, deal sealing) behaviors of a client in order to make subsequent and larger DataCap allocations.


---

**Concern:** *Subsequent allocation calculation is complicated, and takes too long to reach full application request*

For this proposal, we did not plan to change the existing calculation, because it provides a nice scaffold for the progressive allocation of DataCap. This could be addressed in a future proposal.

Please note: the allocations do NOT have to happen weekly. For example, the third allocation may be proposed & approved on a Monday, and by that Wednesday the client may have used 75% of that allocation and could then request the fourth one. This could be proposed and approved on Thursday. *The fourth allocation does not have to wait for the fourth week*

Also, the weekly usage may change (as we have already seen with some LDN's). In the above scenario, if the client is using more per week than anticipated then we could adjust their weekly calculation, which would then allow larger subsequent allocations.

I agree that we could investigate changing this calculation, so that subsequent allocations are even larger and we reach full requested DataCap with fewer allocations. For now, we would like to keep the first allocation as smaller, as the program scales.



---

**Concern:** *2 notaries is too few, should require more at first and then lower threshold*

While not impossible, it is difficult to modify the notary addresses or the threshold of an existing multisig. So, setting the threshold to something like 4 for the first allocation, but then changing the threshold to 2 for subsequent allocations would require additional tooling and increase manual steps, and potential for slow down.

The longest delays so far are happening during the manual steps of the process:
1. After application is submitted, getting 7 notaries to review and approve the application in GitHub.  ~6 weeks
1. After 7 notaries approve, getting the LDN created by root key holders. ~3.5 weeks
1. After LDN is created, getting the first allocation proposed and approved by 4 notaries. Recently, <1 week

This process is designed specifically to address point 1.

We are addressing point 2 by adding root key holders, and we are addressing point 3 by adding more automation (a bot to monitor DataCap remaining and automatically start subsequent requests).

---

**Concern:** *Requiring different notaries for subsequent allocations will become difficult to track and get new notaries each time*

To clarify, the proposal would be that the same 2 notaries could not sign each subsequent request from a single LDN client. 

For example, if notaries A & B sign the first allocation, then two different notaries would need to sign the second (C & D). For the third allocation, it could be A & B again. It is just about getting different notaries for each immediately subsequent allocation.

---


I am very excited to hear more from the community on the calls Tuesday, and if you have additional questions you can post them here in advance. Thank you all for driving the growth of the Filecoin network, and helping us onboard and seal humanity's most important data!

**Comment by pooja** [comment]
Agree with most of what @galen-mcandrew posted above, some other comments/thoughts:

## Notaries as watchpersons, not gatekeepers
I think an important distinction/clarification here is that there's an opportunity for notaries to move from being **gatekeepers** to being **watchpersons**. That is, let's consider moving away from notaries primarily be responsible for who can store data on Filecoin to begin with, which creates a HUGE amount of friction for clients, increases frustration, and reduces adoption rates. Instead, notaries can use audit tooling to help make sure that clients are not doing anything fishy with their allocations -- and if they are, stop clients from getting any more allocations.

I think it's definitely a systemic issue that Estuary, Internet Archive, and Shoah Foundation, for example, have struggled to get DataCap when almost anyone can see that they are legitimate clients who will bring an ENORMOUS amount of value to the Filecoin network. The current process, which makes notaries **gatekeepers**, leads to massive and unnecessary inefficiencies, imho. 

Instead, we could default to a model that has a _little bit_ more trust upfront, but still allows notaries to help protect the network by flagging suspicious activity and correcting for it by preventing suspicious activity from continuing.

We have started to see increasing frustration from clients in the Filecoin community, and may start to lose clients soon. I'd hate to see that happen, and think this is an important paradigm shift (notaries as watchpeople, not gatekeepers) for us to make so we can ensure clients have a good experience with Filecoin Plus in the future.

## 2 notaries for initial signing vs more

If we are open to the idea of a slight evolution in the change of notary role (notaries as watchpersons, not gatekeepers), I think 2 notaries is more than enough for passing an application through. I also think it would be fine for subsequent notaries to be the same as the original notaries. However, we may want to augment with audit tooling so that other notaries can observe the behavior of other notaries and flag suspicious behavior on behalf of other notaries as well.

I think, in general, we'll need to develop good audit tooling that notaries can use to monitor behavior of Fil+ participants and flag suspicious activity for further review/action.

## Addressing deeper systemic concerns

I think there's an opportunity for us to do some deeper thinking about the Fil+ system and iterate towards a more efficient and more performant model for sure! Excited for community members to suggest some new ideas here and for us to continue improving the system overall!

**Comment by XnMatrixSV** [comment]
<html>
<body>
<span data-shimo-docs="[[20,&quot;My&nbsp;opinions&nbsp;are as following.&quot;,&quot;0:\&quot;%23666666\&quot;&quot;],[20,&quot;&nbsp;&quot;,&quot;0:\&quot;%23333333\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;IFBE\&quot;&quot;],[20,&quot;About the root key holder &nbsp;&quot;,&quot;1:\&quot;%23f7f8fa\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;KgYg\&quot;|bullet-id:\&quot;j9Zs\&quot;|bullet:\&quot;circle\&quot;|linespacing:\&quot;150\&quot;&quot;],[20,&quot;To keep this process within 1 week.&quot;,&quot;1:\&quot;%23f7f8fa\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;admb\&quot;|linespacing:\&quot;150\&quot;&quot;],[20,&quot;About notaries&nbsp;&quot;,&quot;1:\&quot;%23f7f8fa\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;IDz8\&quot;|bullet-id:\&quot;PyQp\&quot;|bullet:\&quot;circle\&quot;|linespacing:\&quot;150\&quot;&quot;],[20,&quot;It is recommended to start due diligence on the client with 7 notaries (from more than 2 regions), including at least 1 notary from the same region as the applicant.&nbsp;&nbsp;Only the lead notary and one other notary are required to approve, to&quot;,&quot;1:\&quot;%23f7f8fa\&quot;&quot;],[20,&quot;&nbsp;shorten&nbsp;&quot;,&quot;0:\&quot;%23666666\&quot;&quot;],[20,&quot;the approval time. &nbsp;&quot;,&quot;1:\&quot;%23f7f8fa\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;H7k7\&quot;|linespacing:\&quot;150\&quot;&quot;],[20,&quot;About the applicant&quot;,&quot;1:\&quot;%23f7f8fa\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;Zwrf\&quot;|bullet-id:\&quot;SsrB\&quot;|bullet:\&quot;circle\&quot;|linespacing:\&quot;150\&quot;&quot;],[20,&quot;At the application stage, the applicant shall provide a description of data storage authorized by the data copyright owner.&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;WIF9\&quot;&quot;],[20,&quot;When the applicant has used up more than 75% of the last allocation,  the applicant should provide the summary of the last storage deals, including the miner nodes, the cumulative Datacap used per miner and the corresponding storage proportion.&nbsp;&nbsp;(There should be at least 20 miner nodes at the first usage of DataCap, and the proportion of each node should not exceed 5% at last.)&quot;,&quot;1:\&quot;%23f7f8fa\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;QQeT\&quot;&quot;],[20,&quot;Such as Estuary&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;E5nV\&quot;&quot;],[30,[{&quot;A1&quot;:[40,[[20,&quot;Miner&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot; &quot;],[20,&quot;ID&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;iCc4\&quot;&quot;]],&quot;25:\&quot;s8DA\&quot;&quot;],&quot;B1&quot;:[40,[[20,&quot;Size(GiB)&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;d3Nx\&quot;&quot;]],&quot;25:\&quot;H3qF\&quot;&quot;],&quot;C1&quot;:[40,[[20,&quot;storage &quot;,&quot;1:\&quot;%23f7f8fa\&quot;&quot;],[20,&quot;p&quot;,&quot;1:\&quot;%23f7f8fa\&quot;|26:\&quot;41406726\&quot;&quot;],[20,&quot;roportion&quot;,&quot;1:\&quot;%23f7f8fa\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;VoWY\&quot;&quot;]],&quot;25:\&quot;pRX0\&quot;&quot;],&quot;A2&quot;:[40,[[20,&quot;f010088\n&quot;]],&quot;25:\&quot;KyXh\&quot;&quot;],&quot;B2&quot;:[40,[[20,&quot;2884.455078&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;WRbE\&quot;&quot;],&quot;C2&quot;:[40,[[20,&quot;4.10%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;6GMc\&quot;&quot;],&quot;A3&quot;:[40,[[20,&quot;f01035680\n&quot;]],&quot;25:\&quot;6wKe\&quot;&quot;],&quot;B3&quot;:[40,[[20,&quot;1280&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;6wY1\&quot;&quot;],&quot;C3&quot;:[40,[[20,&quot;1.82%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;kiJW\&quot;&quot;],&quot;A4&quot;:[40,[[20,&quot;f010446\n&quot;]],&quot;25:\&quot;M6AY\&quot;&quot;],&quot;B4&quot;:[40,[[20,&quot;1000&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;pdk3\&quot;&quot;],&quot;C4&quot;:[40,[[20,&quot;1.42%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;aecF\&quot;&quot;],&quot;A5&quot;:[40,[[20,&quot;f0104671\n&quot;]],&quot;25:\&quot;8x2O\&quot;&quot;],&quot;B5&quot;:[40,[[20,&quot;4119.19043&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;Y3Rg\&quot;&quot;],&quot;C5&quot;:[40,[[20,&quot;5.86%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;6MiG\&quot;&quot;],&quot;A6&quot;:[40,[[20,&quot;f010479\n&quot;]],&quot;25:\&quot;ys2D\&quot;&quot;],&quot;B6&quot;:[40,[[20,&quot;5806.625&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;kids\&quot;&quot;],&quot;C6&quot;:[40,[[20,&quot;8.26%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;HA6b\&quot;&quot;],&quot;A7&quot;:[40,[[20,&quot;f01049918\n&quot;]],&quot;25:\&quot;NdKx\&quot;&quot;],&quot;B7&quot;:[40,[[20,&quot;1569.230469&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;p0G0\&quot;&quot;],&quot;C7&quot;:[40,[[20,&quot;2.23%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;PrQH\&quot;&quot;],&quot;A8&quot;:[40,[[20,&quot;f010617\n&quot;]],&quot;25:\&quot;9cq0\&quot;&quot;],&quot;B8&quot;:[40,[[20,&quot;1336&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;2lEp\&quot;&quot;],&quot;C8&quot;:[40,[[20,&quot;1.90%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;yfdJ\&quot;&quot;],&quot;A9&quot;:[40,[[20,&quot;f01240\n&quot;]],&quot;25:\&quot;XK1V\&quot;&quot;],&quot;B9&quot;:[40,[[20,&quot;1548.5625&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;qLe5\&quot;&quot;],&quot;C9&quot;:[40,[[20,&quot;2.20%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;iNsr\&quot;&quot;],&quot;A10&quot;:[40,[[20,&quot;f01247\n&quot;]],&quot;25:\&quot;rNer\&quot;&quot;],&quot;B10&quot;:[40,[[20,&quot;140.25&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;9n3B\&quot;&quot;],&quot;C10&quot;:[40,[[20,&quot;0.20%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;ZpQl\&quot;&quot;],&quot;A11&quot;:[40,[[20,&quot;f01278\n&quot;]],&quot;25:\&quot;aqbY\&quot;&quot;],&quot;B11&quot;:[40,[[20,&quot;380.4140625&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;lJH9\&quot;&quot;],&quot;C11&quot;:[40,[[20,&quot;0.54%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;8cx7\&quot;&quot;],&quot;A12&quot;:[40,[[20,&quot;f0127896\n&quot;]],&quot;25:\&quot;YwlB\&quot;&quot;],&quot;B12&quot;:[40,[[20,&quot;815.25&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;ljlc\&quot;&quot;],&quot;C12&quot;:[40,[[20,&quot;1.16%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;WdBo\&quot;&quot;],&quot;A13&quot;:[40,[[20,&quot;f0135078\n&quot;]],&quot;25:\&quot;S5o0\&quot;&quot;],&quot;B13&quot;:[40,[[20,&quot;2768&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;mrob\&quot;&quot;],&quot;C13&quot;:[40,[[20,&quot;3.94%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;yfTO\&quot;&quot;],&quot;A14&quot;:[40,[[20,&quot;f014409\n&quot;]],&quot;25:\&quot;ukRr\&quot;&quot;],&quot;B14&quot;:[40,[[20,&quot;56&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;zuKC\&quot;&quot;],&quot;C14&quot;:[40,[[20,&quot;0.08%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;Oard\&quot;&quot;],&quot;A15&quot;:[40,[[20,&quot;f014768\n&quot;]],&quot;25:\&quot;Q7ta\&quot;&quot;],&quot;B15&quot;:[40,[[20,&quot;1795.146484&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;jBPg\&quot;&quot;],&quot;C15&quot;:[40,[[20,&quot;2.55%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;mF8y\&quot;&quot;],&quot;A16&quot;:[40,[[20,&quot;f0157535\n&quot;]],&quot;25:\&quot;M0Hz\&quot;&quot;],&quot;B16&quot;:[40,[[20,&quot;408&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;Xs0v\&quot;&quot;],&quot;C16&quot;:[40,[[20,&quot;0.58%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;SzbX\&quot;&quot;],&quot;A17&quot;:[40,[[20,&quot;f015927\n&quot;]],&quot;25:\&quot;H6ei\&quot;&quot;],&quot;B17&quot;:[40,[[20,&quot;784&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;2jKn\&quot;&quot;],&quot;C17&quot;:[40,[[20,&quot;1.12%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;TUGM\&quot;&quot;],&quot;A18&quot;:[40,[[20,&quot;f0165400\n&quot;]],&quot;25:\&quot;MVjx\&quot;&quot;],&quot;B18&quot;:[40,[[20,&quot;96&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;XOfZ\&quot;&quot;],&quot;C18&quot;:[40,[[20,&quot;0.14%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;Wfew\&quot;&quot;],&quot;A19&quot;:[40,[[20,&quot;f019104\n&quot;]],&quot;25:\&quot;2atJ\&quot;&quot;],&quot;B19&quot;:[40,[[20,&quot;344&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;1XIk\&quot;&quot;],&quot;C19&quot;:[40,[[20,&quot;0.49%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;apFn\&quot;&quot;],&quot;A20&quot;:[40,[[20,&quot;f019551\n&quot;]],&quot;25:\&quot;M4AC\&quot;&quot;],&quot;B20&quot;:[40,[[20,&quot;1827.689453&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;5Fu9\&quot;&quot;],&quot;C20&quot;:[40,[[20,&quot;2.60%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;ULOf\&quot;&quot;],&quot;A21&quot;:[40,[[20,&quot;f020378\n&quot;]],&quot;25:\&quot;G12l\&quot;&quot;],&quot;B21&quot;:[40,[[20,&quot;352.75&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;E8V3\&quot;&quot;],&quot;C21&quot;:[40,[[20,&quot;0.50%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;ibis\&quot;&quot;],&quot;A22&quot;:[40,[[20,&quot;f020385\n&quot;]],&quot;25:\&quot;lx9d\&quot;&quot;],&quot;B22&quot;:[40,[[20,&quot;784&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;FIKT\&quot;&quot;],&quot;C22&quot;:[40,[[20,&quot;1.12%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;8vWI\&quot;&quot;],&quot;A23&quot;:[40,[[20,&quot;f022142\n&quot;]],&quot;25:\&quot;j1NZ\&quot;&quot;],&quot;B23&quot;:[40,[[20,&quot;1599.274414&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;KCwu\&quot;&quot;],&quot;C23&quot;:[40,[[20,&quot;2.28%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;AT6E\&quot;&quot;],&quot;A24&quot;:[40,[[20,&quot;f022163\n&quot;]],&quot;25:\&quot;WsK4\&quot;&quot;],&quot;B24&quot;:[40,[[20,&quot;1782&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;GYZL\&quot;&quot;],&quot;C24&quot;:[40,[[20,&quot;2.54%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;YY98\&quot;&quot;],&quot;A25&quot;:[40,[[20,&quot;f022352\n&quot;]],&quot;25:\&quot;9W5z\&quot;&quot;],&quot;B25&quot;:[40,[[20,&quot;1190.569336&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;kB1i\&quot;&quot;],&quot;C25&quot;:[40,[[20,&quot;1.69%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;ckMA\&quot;&quot;],&quot;A26&quot;:[40,[[20,&quot;f023467\n&quot;]],&quot;25:\&quot;tWd6\&quot;&quot;],&quot;B26&quot;:[40,[[20,&quot;1986.25&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;KY8B\&quot;&quot;],&quot;C26&quot;:[40,[[20,&quot;2.83%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;r4X5\&quot;&quot;],&quot;A27&quot;:[40,[[20,&quot;f023971\n&quot;]],&quot;25:\&quot;jsZE\&quot;&quot;],&quot;B27&quot;:[40,[[20,&quot;601&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;pgBa\&quot;&quot;],&quot;C27&quot;:[40,[[20,&quot;0.86%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;cCbT\&quot;&quot;],&quot;A28&quot;:[40,[[20,&quot;f024184\n&quot;]],&quot;25:\&quot;1tHe\&quot;&quot;],&quot;B28&quot;:[40,[[20,&quot;1186&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;e1fH\&quot;&quot;],&quot;C28&quot;:[40,[[20,&quot;1.69%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;9lrc\&quot;&quot;],&quot;A29&quot;:[40,[[20,&quot;f02576\n&quot;]],&quot;25:\&quot;XF6A\&quot;&quot;],&quot;B29&quot;:[40,[[20,&quot;3325.910156&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;IwyK\&quot;&quot;],&quot;C29&quot;:[40,[[20,&quot;4.73%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;2MHK\&quot;&quot;],&quot;A30&quot;:[40,[[20,&quot;f02606\n&quot;]],&quot;25:\&quot;gabh\&quot;&quot;],&quot;B30&quot;:[40,[[20,&quot;707.5&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;v9ga\&quot;&quot;],&quot;C30&quot;:[40,[[20,&quot;1.01%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;MrZa\&quot;&quot;],&quot;A31&quot;:[40,[[20,&quot;f02620\n&quot;]],&quot;25:\&quot;T8cf\&quot;&quot;],&quot;B31&quot;:[40,[[20,&quot;1019&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;OMCu\&quot;&quot;],&quot;C31&quot;:[40,[[20,&quot;1.45%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;CoZT\&quot;&quot;],&quot;A32&quot;:[40,[[20,&quot;f030379\n&quot;]],&quot;25:\&quot;Rurk\&quot;&quot;],&quot;B32&quot;:[40,[[20,&quot;994&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;8UHo\&quot;&quot;],&quot;C32&quot;:[40,[[20,&quot;1.41%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;624c\&quot;&quot;],&quot;A33&quot;:[40,[[20,&quot;f03488\n&quot;]],&quot;25:\&quot;knsQ\&quot;&quot;],&quot;B33&quot;:[40,[[20,&quot;1386&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;cAyT\&quot;&quot;],&quot;C33&quot;:[40,[[20,&quot;1.97%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;x6Ol\&quot;&quot;],&quot;A34&quot;:[40,[[20,&quot;f0406322\n&quot;]],&quot;25:\&quot;OYT9\&quot;&quot;],&quot;B34&quot;:[40,[[20,&quot;4128&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;IO5k\&quot;&quot;],&quot;C34&quot;:[40,[[20,&quot;5.87%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;Ie2l\&quot;&quot;],&quot;A35&quot;:[40,[[20,&quot;f0406703\n&quot;]],&quot;25:\&quot;xrgJ\&quot;&quot;],&quot;B35&quot;:[40,[[20,&quot;2232&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;Xxc0\&quot;&quot;],&quot;C35&quot;:[40,[[20,&quot;3.18%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;EHCy\&quot;&quot;],&quot;A36&quot;:[40,[[20,&quot;f0440429\n&quot;]],&quot;25:\&quot;oEty\&quot;&quot;],&quot;B36&quot;:[40,[[20,&quot;184&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;IDxE\&quot;&quot;],&quot;C36&quot;:[40,[[20,&quot;0.26%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;17LM\&quot;&quot;],&quot;A37&quot;:[40,[[20,&quot;f0492030\n&quot;]],&quot;25:\&quot;FECG\&quot;&quot;],&quot;B37&quot;:[40,[[20,&quot;342.75&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;cMWw\&quot;&quot;],&quot;C37&quot;:[40,[[20,&quot;0.49%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;LUaC\&quot;&quot;],&quot;A38&quot;:[40,[[20,&quot;f058369\n&quot;]],&quot;25:\&quot;0TlD\&quot;&quot;],&quot;B38&quot;:[40,[[20,&quot;611.5634766&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;yypI\&quot;&quot;],&quot;C38&quot;:[40,[[20,&quot;0.87%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;2RCA\&quot;&quot;],&quot;A39&quot;:[40,[[20,&quot;f062353\n&quot;]],&quot;25:\&quot;j93u\&quot;&quot;],&quot;B39&quot;:[40,[[20,&quot;144&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;HsZq\&quot;&quot;],&quot;C39&quot;:[40,[[20,&quot;0.20%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;SedH\&quot;&quot;],&quot;A40&quot;:[40,[[20,&quot;f066596\n&quot;]],&quot;25:\&quot;USBx\&quot;&quot;],&quot;B40&quot;:[40,[[20,&quot;1488.441406&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;lHUd\&quot;&quot;],&quot;C40&quot;:[40,[[20,&quot;2.12%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;dm8E\&quot;&quot;],&quot;A41&quot;:[40,[[20,&quot;f0678914\n&quot;]],&quot;25:\&quot;VZ84\&quot;&quot;],&quot;B41&quot;:[40,[[20,&quot;1148.5&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;GsdJ\&quot;&quot;],&quot;C41&quot;:[40,[[20,&quot;1.63%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;TFKP\&quot;&quot;],&quot;A42&quot;:[40,[[20,&quot;f0694396\n&quot;]],&quot;25:\&quot;RLSS\&quot;&quot;],&quot;B42&quot;:[40,[[20,&quot;2741.572266&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;KXze\&quot;&quot;],&quot;C42&quot;:[40,[[20,&quot;3.90%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;PXVp\&quot;&quot;],&quot;A43&quot;:[40,[[20,&quot;f0773157\n&quot;]],&quot;25:\&quot;MBVi\&quot;&quot;],&quot;B43&quot;:[40,[[20,&quot;2217.0625&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;57iM\&quot;&quot;],&quot;C43&quot;:[40,[[20,&quot;3.15%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;9Rsl\&quot;&quot;],&quot;A44&quot;:[40,[[20,&quot;f08399\n&quot;]],&quot;25:\&quot;SkrV\&quot;&quot;],&quot;B44&quot;:[40,[[20,&quot;2777&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;6F41\&quot;&quot;],&quot;C44&quot;:[40,[[20,&quot;3.95%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;Y7zr\&quot;&quot;],&quot;A45&quot;:[40,[[20,&quot;f08403\n&quot;]],&quot;25:\&quot;yIBp\&quot;&quot;],&quot;B45&quot;:[40,[[20,&quot;938.5078125&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;o3Qe\&quot;&quot;],&quot;C45&quot;:[40,[[20,&quot;1.34%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;S7Aw\&quot;&quot;],&quot;A46&quot;:[40,[[20,&quot;f0875769\n&quot;]],&quot;25:\&quot;Jea0\&quot;&quot;],&quot;B46&quot;:[40,[[20,&quot;1152&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;hIpJ\&quot;&quot;],&quot;C46&quot;:[40,[[20,&quot;1.64%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;zunG\&quot;&quot;],&quot;A47&quot;:[40,[[20,&quot;f09848\n&quot;]],&quot;25:\&quot;PbEJ\&quot;&quot;],&quot;B47&quot;:[40,[[20,&quot;4296&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;MdQH\&quot;&quot;],&quot;C47&quot;:[40,[[20,&quot;6.11%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;]],&quot;25:\&quot;TdLt\&quot;&quot;],&quot;A48&quot;:[40,[[20,&quot;T&quot;],[20,&quot;otal&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;LSQ7\&quot;&quot;]],&quot;25:\&quot;Jt3o\&quot;&quot;],&quot;B48&quot;:[40,[[20,&quot;70274.46484&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;V76h\&quot;&quot;]],&quot;25:\&quot;xpK8\&quot;&quot;],&quot;C48&quot;:[40,[[20,&quot;1&quot;],[20,&quot;00%&quot;,&quot;26:\&quot;41406726\&quot;&quot;],[20,&quot;\n&quot;,&quot;24:\&quot;in0Z\&quot;&quot;]],&quot;25:\&quot;Pl3e\&quot;&quot;]},[[10,48]],[[10,1,&quot;3:148&quot;],[10,1,&quot;3:258&quot;],[10,1,&quot;3:187&quot;]]],&quot;25:\&quot;bQkDJy\&quot;&quot;]]"><p line="IFBE" class="ql-long-41406726" style="line-height: 1.7;margin-bottom: 0pt;margin-top: 0pt;font-size: 11pt;color: #494949;"><span class="ql-author-41406726" style="color: rgb(102, 102, 102);color: #666666;">My&nbsp;opinions&nbsp;are as following.</span><span class="ql-author-41406726" style="color: rgb(51, 51, 51);color: #333333;">&nbsp;</span></p><ul><li class="" style="margin-bottom: 0pt;margin-top: 0pt;font-size: 11pt;color: #494949;line-height: 150%;"><span class="ql-author-41406726" style="background-color: rgb(247, 248, 250);">About the root key holder &nbsp;</span></li></ul><p line="admb" linespacing="150" class="ql-long-41406726" style="line-height: 150%;margin-bottom: 0pt;margin-top: 0pt;font-size: 11pt;color: #494949;"><span class="ql-author-41406726" style="background-color: rgb(247, 248, 250);">To keep this process within 1 week.</span></p><ul><li class="" style="margin-bottom: 0pt;margin-top: 0pt;font-size: 11pt;color: #494949;line-height: 150%;"><span class="ql-author-41406726" style="background-color: rgb(247, 248, 250);">About notaries&nbsp;</span></li></ul><p line="H7k7" linespacing="150" class="ql-long-41406726" style="line-height: 150%;margin-bottom: 0pt;margin-top: 0pt;font-size: 11pt;color: #494949;"><span class="ql-author-41406726" style="background-color: rgb(247, 248, 250);">It is recommended to start due diligence on the client with 7 notaries (from more than 2 regions), including at least 1 notary from the same region as the applicant.&nbsp;&nbsp;Only the lead notary and one other notary are required to approve, to</span><span class="ql-author-41406726" style="color: rgb(102, 102, 102);color: #666666;">&nbsp;shor</span><span style="color: rgb(102, 102, 102);color: #666666;">ten</span><span class="ql-author-41406726" style="color: rgb(102, 102, 102);color: #666666;">&nbsp;</span><span class="ql-author-41406726" style="background-color: rgb(247, 248, 250);">the approval time. &nbsp;</span></p><ul><li class="" style="margin-bottom: 0pt;margin-top: 0pt;font-size: 11pt;color: #494949;line-height: 150%;"><span class="ql-author-41406726" style="background-color: rgb(247, 248, 250);">About the applicant</span></li></ul><p line="WIF9" style="" class="ql-long-41406726"><span class="ql-author-41406726" style="">At the application stage, the applicant shall provide a description of data storage authorized by the data copyright owner.</span></p><p line="QQeT" class="ql-long-41406726" style="line-height: 1.7;margin-bottom: 0pt;margin-top: 0pt;font-size: 11pt;color: #494949;"><span class="ql-author-41406726" style="background-color: rgb(247, 248, 250);">When the applicant has used up more than 75% of the last allocation,  the applicant should provide the summary of the last storage deals, including the miner nodes, the cumulative Datacap used per miner and the corresponding storage proportion.&nbsp;&nbsp;(There should be at least 20 miner nodes at the first usage of DataCap, and the proportion of each node should not exceed 5% at last.)</span><span style="background-color: rgb(247, 248, 250);"></span></p><p line="E5nV" class="ql-long-41406726" style="line-height: 1.7;margin-bottom: 0pt;margin-top: 0pt;font-size: 11pt;color: #494949;"><span class="ql-author-41406726" style="">Such as Estuary</span></p>

Miner ID | Size(GiB) | storage proportion
-- | -- | --
f010088 | 2884.455078 | 4.10%
f01035680 | 1280 | 1.82%
f010446 | 1000 | 1.42%
f0104671 | 4119.19043 | 5.86%
f010479 | 5806.625 | 8.26%
f01049918 | 1569.230469 | 2.23%
f010617 | 1336 | 1.90%
f01240 | 1548.5625 | 2.20%
f01247 | 140.25 | 0.20%
f01278 | 380.4140625 | 0.54%
f0127896 | 815.25 | 1.16%
f0135078 | 2768 | 3.94%
f014409 | 56 | 0.08%
f014768 | 1795.146484 | 2.55%
f0157535 | 408 | 0.58%
f015927 | 784 | 1.12%
f0165400 | 96 | 0.14%
f019104 | 344 | 0.49%
f019551 | 1827.689453 | 2.60%
f020378 | 352.75 | 0.50%
f020385 | 784 | 1.12%
f022142 | 1599.274414 | 2.28%
f022163 | 1782 | 2.54%
f022352 | 1190.569336 | 1.69%
f023467 | 1986.25 | 2.83%
f023971 | 601 | 0.86%
f024184 | 1186 | 1.69%
f02576 | 3325.910156 | 4.73%
f02606 | 707.5 | 1.01%
f02620 | 1019 | 1.45%
f030379 | 994 | 1.41%
f03488 | 1386 | 1.97%
f0406322 | 4128 | 5.87%
f0406703 | 2232 | 3.18%
f0440429 | 184 | 0.26%
f0492030 | 342.75 | 0.49%
f058369 | 611.5634766 | 0.87%
f062353 | 144 | 0.20%
f066596 | 1488.441406 | 2.12%
f0678914 | 1148.5 | 1.63%
f0694396 | 2741.572266 | 3.90%
f0773157 | 2217.0625 | 3.15%
f08399 | 2777 | 3.95%
f08403 | 938.5078125 | 1.34%
f0875769 | 1152 | 1.64%
f09848 | 4296 | 6.11%
Total | 70274.46484 | 100%

</span>
</body>
</html>

**Comment by MegTei** [comment]
Pooja makes some clear and compelling arguments, especially notaries acting as ‘guardians’ rather than bottlenecks. To consider this, it is worth revisiting the original intent, was it largely to serve Filecoins mission to store humanities most important data? An audit role over gatekeeper will still serve this mission. 

Its worth addressing Molly’s question as well, what are the root causes of why its difficult for Notaries and RKH to organise themselves? 
Comms and tools are one aspect, and the automated new tooling talked through this morning will definitely add value.  
Again, its worth revisiting why these roles exist in the first place and why do we self elect to have a seat at this table? 

Perspectives will naturally vary but one common thread should be that we agree to find time to enable this network. 

**I propose that Notaries and RKH sign up to ‘Service levels’, or time boxing.**  For e.g. Notaries could nominate levels of ‘service’ or commitment they can provide. Leads are more active, general Notaries less involved. The guardrails could be something like, leads agree to check in daily and general every couple days. We should aim to turnaround a LDN in 5 days. General requests in a few days. 

Is motivation and incentive an issue for these roles? It isn’t for Holon, our motivations are that the tide raises all boats, that is its holistic, what we achieve through this DAO enables what ecosystem businesses/ entrepreneurs can deliver in their business plans. Everyone is different however and perhaps applying some level of meritocracy may bring to life the input at various levels.

**Comment by pooja** [comment]
I think @MegTei's idea of service-level commitments is a good one!

One nudge here too is: if we wanted notaries/LDNs to be able to **approve applications and make the first allocation in less than 1 day**, what would we have to do to improve the system? From a client experience perspective, this really is what we need to aim for imo and from talking to some of our clients! Going from 6-8 weeks to <1 week is good, but it's still not good enough from a user perspective, so I think we may need to be creative and open-minded as a governance community with possible solutions to help us deliver that order of magnitude improvement to Filecoin clients

**Comment by dkkapur** [comment]
Hi folks - thanks for engaging actively on various aspects of this proposal + the LDN process! Great to see the different ideas, perspectives, and suggestions for ways in which we can make the LDN process, and Fil+ overall, more effective! Just wanted to share some points from the discussions at the governance calls earlier this week (recordings of both the calls are here: https://www.youtube.com/watch?v=o0nPBRM-aMQ).
- Goals of the LDN flow and this proposal
  - Filecoin Plus aims to make the Filecoin Network more productive, by reducing the friction for clients to onboard their datasets on the network and better aligning the incentives of storage providers with the needs of clients. The LDN flow was created to support clients who had massive datasets that extended beyond the individual
  - the current LDN flow was designed to be executed on as a scoped initial experiment / v1 of how we can support clients at scale (with initial limits at 50PiB). As a community, we're hoping to continue learning and evolving this and other processes to make Filecoin a more productive network
  - Time to DataCap via the LDN process today is in the order of several weeks for the first allocation, and still at several days for subsequent allocations. If clients are to consider using Filecoin at scale / for larger production use cases, waiting several days+ is not a viable path for most organizations looking to onboard data at scale. The goal of this proposal is to start pushing the initial LDN process to the next level in terms of serving real client demand at scale by drastically reducing the time it takes legitimate clients to be unblocked and begin onboarding their datasets to Filecoin
- Feedback/suggestions to the process changes proposed 
  - Allocation calculations - increasing the DataCap limits for each subsequent allocations over time
    - several ways in which this aspect can be improved have been suggested in this discussion - thanks to each of you who have suggested interesting/new ways to do this. These include continuing to scale subsequent allocations at a much higher than currently, or increasing them for many more allocations than the current first 3, or changing them to be based on actual on-chain data onboarding rates, etc. 
    - folks interested in alternate ways of doing this or with ideas on how this can improve should review the suggestions made in this discussion and engage accordingly! The appropriate next step for ways to actually change this process is to then suggest this as a separate proposal (new Issue/Discussion) that the community can then collaborate on.
  - Threshold of signature on the multisig - whether or not the proposed 2 is appropriate/safe/enough, and notary signature rotation
    - as we think about continuing to evolve the Filecoin Plus program into a much more holistic governance entity for the network / decentralized autonomous organization (DAO), the role of the Notary will continue to be pivotal but also need to shift to becoming more of a watchperson, where Notaries and other relevant stakeholders in the ecosystem work together to govern the network based on data available about entities in the network 
    - enabling clients to get started with dealmaking quickly with limited initial DataCap allocations gives them a chance to prove that they will be productive on the network, and gives Notaries a chance to not only do due diligence on these clients, but also start collecting data ASAP on their on-chain behavior and dealmaking actions. This then leads to a much more holistic understanding of legitimate clients on the network and ensures productive clients are enabled quicker and continue to be supported for the long term. This also helps the community make better decisions on governance processes and policies, such as improving the rubric for Notary elections
    - additional follow-up proposals were also suggested with regards to introducing regional requirements for signing notaries approving messages on the multi-sig, i.e., ensuring at least 1 approve is from the region the client is in. This is definitely interesting in terms of tapping into the regional expertise of notaries, however, applications are still going to be completely public and open for review by all notaries and community members to diligence on, so we can continue to leverage our distributed/global perspective in the community. 
    - this is still another step in the direction of improving support for clients with large datasets. As a community, we should continue to expect to iterate - learn from the feedback we hear from respective stakeholders and continue proposing ways in which Fil+ can be more effective. Biasing on the side of collecting data makes sense for us when the network and the program are still so young
- Additional discussion on aspects of the Filecoin Plus, i.e., MegTei's proposal above on "service levels"
  - Great ideas suggested as parallel or independent avenues to explore in making Fil+ more effective/efficient; would be awesome to see these proposed individually for the community to discuss.

**Comment by galen-mcandrew** [comment]
Some really great ideas and discussion here, with some really exciting potential directions for the future! 

We've synthesized the discussion, and we'd like to move towards resolution. As a reminder of the original goal and intent:

**Our primary goal with this proposal is to decrease the initial time to first DataCap for large dataset clients. We can accomplish this by adding automation and reducing the initial notary threshold.**

To recap the timeline thus far:
* **August 3rd Governance Call:** Discussion around LDN slow-down, started discussing areas of improvement
* **August 13th Opened Issue 217:** Initial proposal was designed by Deep and Galen, with input from Filecoin Foundation, Protocol Labs, Notaries, Clients, and technical developers
* **August 17th Governance Call:** Presented Issue 217, discussed comments and questions; continued discussion directly in Issue
* **August 31st Governance Call:** Presented Issue 217 again, synthesized and addressed concerns, more open discussion


After these discussions, we feel that this proposal is **safe to move forward** as an experiment for *100 PiB of DataCap*. Steps for rolling out:
* **September 14th Governance Call:** Demo end-to-end process, including new tooling. Close Issue 217. 
* **Week of September 14th:** Begin rolling existing LDN applications to new process, including creating new multisig notaries for existing approved LDN's (this is much simpler for changing the signers and threshold, compared to adjusting the existing multisig).

During this experimental period, the community will continue to monitor and flag concerns, as well as surface new ideas for improvement. Some specific things the governance team will be watching for:
* **Time to DataCap for LDN clients:** both new and existing, first and subsequent allocations
* **Notary engagement with clients:** Slack and GitHub discussions, diligence on deal behavior
* **New LDN applications:** net new, paying special attention to number of subsequent allocation requests
* **Technical improvements:** bugs, velocity impact, audit usefulness

If you are a notary and would like to **opt-out** of this experiment, please comment below with your address. We will not include you in the LDN multisigs that are created for these large dataset clients. This will *not* impact your direct client onboarding process.

We hope that this 100 PiB experiment is successful in helping us unblock and onboard some key strategic partners at this time in the network! Thank you for all the discussion and ideation. We have heard some other great ideas from the community during this process, and we will be opening up new discussion threads to explore those topics!

**Comment by dannyob** [comment]
This is with two signatories, right? As someone who brought that up originally, think this is the right way to proceed -- an experiment to see what issues (if any) emerge. Also good to hear about new tooling! If we're heading towards the watchperson role, it'd be great to see some more tools to let us learn what actions datacap recipients have made after receiving their allotment.

Shall we continue noting on the experiment in this issue, or start new ones?

**Comment by dkkapur** [comment]
@dannyob I believe so - I think noting takeaways from the experiment phase can happen in this Issue, but actual tactical follow-ups or amendments (+ respective discussion) as a result of our observation should happen in a separate issue.
